### DISCUSSION TITLE
Accordian component

### DISCUSSION THREAD

**Discussion by ProPhycient5** [saadeghi/daisyui#495] [Ideas / feature requests]
Are you guys planning to build dynamic accordion component ?

**Comment by saadeghi** [comment]
@ProPhycient5 What do you mean by "dynamic"?

**Reply by ProPhycient5** [reply]
Having control of the toggle like click as eventlistener.

**Reply by ProPhycient5** [reply]
I can better convey my opinion on this link (https://kimia-ui.vercel.app/components/accordion).
I hope after visiting this link, you get what I mean to say.

**Reply by saadeghi** [reply]
[collapse](https://daisyui.com/components/collapse/) works with `focus` so you can listen to `focus` and `blur` events:  
https://svelte.dev/repl/75765ef76b2941db9012b4d8fb18228b?version=3.46.4  

You can also use [the ones that work with checkbox](https://daisyui.com/components/collapse/#collapse-with-checkbox) and listen to the checkbox changes

**Reply by ProPhycient5** [reply]
Yeah I checked, in this link (https://svelte.dev/repl/75765ef76b2941db9012b4d8fb18228b?version=3.46.4) , accordion header is not toggle type, whereas Collapse with checkbox works as I wanted and how accordion is expected to work.
